### PR TITLE
Loop

### PR DESCRIPTION
**Pull request by kkobierski**
Works for either firm or lawyer rankings at the moment.

### CODE REVIEW

File: data/firm_111.csv [hunk 1]
```diff
@@ -2,95 +2,95 @@ Ranked Departments,6,Banking & Finance,16083,Africa-wide,Band 4,2,111
...
 Ranked Departments,996,Projects & Energy,16083,Africa-wide,Band 3,2,111
-Ranked Departments,24,Energy & Natural Resources,16084,Asia-Pacific Region,Recognised Practitioner,2,111
-Ranked Departments,1486,Banking & Finance: Corporate Finance,23,Australia,Band 4,2,111
-Ranked Departments,1483,Banking & Finance: Property Finance,23,Australia,Band 3,2,111
-Ranked Departments,354,Corporate/M&A,23,Australia,Recognised Practitioner,2,111
-Ranked Departments,467,Dispute Resolution,23,Australia,Recognised Practitioner,2,111
-Ranked Departments,24,Energy & Natural Resources,23,Australia,Band 5,2,111
-Ranked Departments,852,General Business Law,25,Azerbaijan,Band 1,2,111
-Ranked Departments,852,General Business Law,30,Barbados,Recognised Practitioner,2,111
+Ranked Departments,24,Energy & Natural Resources,16084,Asia-Pacific Region,Recognised Practitioner,2,9152
```
Comment (dwabece) [inline data/firm_111.csv]: why to keep that file at all?

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"""
+settings module for the app
+"""
+
+FIRMS_LIST = [111, 7821, 126396, 250204]
+
+PUBLICATION_ID = 2
+
+#
```
Comment (dwabece) [inline settings.py:9]: why to keep that?

Comment (kkobierski) [reply settings.py:9]: PUBLICATION_ID is needed to create a n url

File: data/firm_111.csv (hunk 1 above)
Comment (kkobierski) [reply data/firm_111.csv]: I don't want to keep it. Just can't get rid of it.

**Pull request merged**